kore/src/Kore/Repl/Interpreter.hs

```diff
@@ -633,10 +633,26 @@ printNotFound = putStrLn' "Variable or index not found"
 putStrLn' :: MonadWriter String m => String -> m ()
 putStrLn' str = tell $ str <> "\n"
 
-showDotGraph :: Graph gr => gr nl el -> IO ()
-showDotGraph =
+showDotGraph :: Int -> GraphType -> IO ()
+showDotGraph len =
     (flip Graph.runGraphvizCanvas') Graph.Xlib
-        . Graph.graphToDot Graph.nonClusteredParams
+        . Graph.graphToDot params
+  where
+    params = Graph.nonClusteredParams
+        { Graph.fmtEdge = \(_, _, l) ->
+            [Graph.textLabel (ruleIndex len l)]
+        }
+    ruleIndex lth lbl =
+        case listToMaybe . toList $ lbl of
+            Nothing -> "Simpl/RD"
+            Just rule -> LT.pack
```

Comment:
If we changed the Identifier attribute to also use Lazy Text, we could just use that version of Text in this module and import it as `Text` rather than using both. It should really not matter much, so I'd rather do that.